src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -38,13 +38,13 @@ SQL_COMMENT: '/*' ~'!' (.)*? '*/' -> skip;
 
 SQL_LINE_COMMENT: ('#' | '--') (~'\n')* ('\n' | EOF) -> skip;
 
-STRING_LITERAL: TICK ('\\\'' | '\'\'' | ~('\''))* TICK;
+STRING_LITERAL: [a-zA-Z]? TICK ('\\\'' | '\'\'' | ~('\''))* TICK;
```

Comment:
isn't this a bit too relaxed? What else can come here other than b and x? maybe o for octal?